org.sonarlint.eclipse.ui/plugin.xml

```diff
@@ -664,7 +663,7 @@
           commandId="org.sonarlint.eclipse.ui.internal.command.ExcludeCommand">
            <enabledWhen>
 	        <with variable="activeMenuSelection">
-	          <iterate ifEmpty="false">
+	          <iterate ifEmpty="false" operator="or">
 	            <or>
```

Comment:
What is the point of this "or". Shouldn't be a and?

Reply:
In fact I think it's a no-op in this case, since there is only on adapt being done. The exclusion test is part of the adapt.

Reply:
Arf, you're right, I was confused by the indentation ;)